cache/lib/cache_web/api/schemas/multipart_upload.ex

```diff
@@ -0,0 +1,41 @@
+defmodule CacheWeb.API.Schemas.StartMultipartUploadResponse do
```

Comment:
nit: if we want to group the schemas, then I wouldn't be putting two modules into a single file but instead would do:
```
schems/multipart_upload/start_multipart_upload_response.ex

defmodule CacheWeb.API.Schemas.MultipartUpload.StartMultipartUploadResponse


...
```